src/components/screens/Service.tsx

```diff
@@ -109,7 +117,12 @@ export const Service = ({
     </DiagonalGradient>
   ) : (
     <>
-      <WrapperWrap spaceBetween>{tiles?.map(renderItem)}</WrapperWrap>
+      <WrapperWrap
+        style={[styles.tilesContainer, shouldCenter && styles.centerLastRow]}
+        spaceBetween
```

Comment:
that wrapper has props already:

![image](https://github.com/user-attachments/assets/3397f11a-1a53-44de-8368-612cccfe7150)

so we should decide wether to have `spaceBetween` or `spaceAround` applied based on `shouldCenter` instead of adding a new individual component style, right!?

Reply:
Yes, you're absolutely right. I will update the code.